### ISSUE TITLE
funcion causes sass-lint to hang during parsing

### ISSUE THREAD

**Issue by jonny-improbable** [sasstools/sass-lint#732]
The following function causes `sass-lint` 1.7.0 to hang when run with the default rule-set:

``` sass
@function closestEvenNumber($number)
  @if ($number % 2 == 0px)
    @return $number
  @else
    @return ($number + 1px)
```

If you write the above our to `bad-function.sass` then the following command invocation will hang and peg the `node` process at 100% CPU:

```
sass-lint ./bad-function.sass
```

**Comment by DanPurdy** [comment]
It's actually a problem with the AST we use, `gonzales-pe`, I'll file an issue there.

**Comment by DanPurdy** [comment]
Oops didn't mean to close it..

**Comment by bgriffith** [comment]
This is fixed in the newly released version v1.11.0. Please let me know if you still have issues. Thanks!
